db/external_sst_file_ingestion_job.cc

```diff
@@ -622,7 +622,10 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
     if (key.sequence != 0) {
       return Status::Corruption("External file has non zero sequence number");
     }
-    file_to_ingest->smallest_internal_key.SetFrom(key);
+
+    SequenceNumber assigned_seqno = versions_->LastSequence();
+
+    file_to_ingest->smallest_internal_key.Set(key.user_key, assigned_seqno, ValueType::kTypeValue);
```

Comment:
Shouldn't the last arg be key.type and not ValueType::kTypeValue??

Reply:
hmm, I Could not find the key.type, the value type is defined here:
https://github.com/facebook/rocksdb/blob/711881bc257d761fd44924686069cb5a0cea580b/db/dbformat.h